samples/src/Microsoft.Azure.IIoT.App/src/Pages/Endpoints.razor

```diff
@@ -72,39 +69,31 @@
         </tr>
     </thead>
     <tbody>
-        @foreach (var endpoint in PagedendpointList.Results)
-        {
+        @foreach (var endpoint in PagedendpointList.Results) {
             <tr class="@GetEndpointVisibilityString(endpoint)">
                 @if (endpoint.EndpointModel.EndpointState == EndpointConnectivityState.Ready ||
-               endpoint.EndpointModel.EndpointState == EndpointConnectivityState.Unauthorized)
-                {
-                    <td class="hover-text width-large">
-                        <div>
-                            @{ string discover = DiscovererId ?? RegistryHelper.PathAll; }
-                            @{ string application = ApplicationId ?? RegistryHelper.PathAll; }
-                            @{ string supervisor = SupervisorId ?? RegistryHelper.PathAll; }
-                            <a href='browser/1/@discover/@application/@supervisor/@endpoint.EndpointModel.Registration.Id'>
-                                @endpoint.EndpointModel.Registration.Endpoint.Url
-                            </a>
-                        </div>
-                    </td>
-                }
-                else
-                {
-                    <td class="hover-text width-large">
-                        <div>
-                            @endpoint.EndpointModel.Registration.Endpoint.Url
-                        </div>
-                    </td>
-                }
-                <td class="hover-text width-small">
-                    <div>
-                        @endpoint.EndpointModel.Registration.Endpoint.SecurityMode
-                    </div>
-                </td>
-                <td class="hover-text width-small">
+             endpoint.EndpointModel.EndpointState == EndpointConnectivityState.Unauthorized) {
+            <td class="hover-text width-large">
+                <div>
+                    @{ string discover = DiscovererId ?? RegistryHelper.PathAll; }
+                    @{ string application = ApplicationId ?? RegistryHelper.PathAll; }
+                    @{ string supervisor = SupervisorId ?? RegistryHelper.PathAll; }
+                    <a href='browser/1/@discover/@application/@supervisor/@endpoint.EndpointModel.Registration.Id'>
```

Comment:
Why not just one @{ ... } block for the three lines?

Reply:
no fixed